### ISSUE TITLE
Playback failed for 4K HDR Atmos movie

### ISSUE THREAD

**Issue by Jelmerb** [plexinc/plex-for-kodi#214]
## Description:
Playback failes for 4K HDR Atmos movie while other movie with same attributes stream perfectly.
Somehow Plex Media Server forced to transcode the file. I know my NAS is not nearly powerfull enough to transcode, so no surprise that failes. But Kodi should play the file just fine. What is wrong?

I start the movie at 20:05:20 

 [Log](https://paste.kodi.tv/oreyehivag)

## Details:
- Add-on version: 0.1.6
- Kodi version: 17.6
- OS Platform: Android 
- Hardware: NVIDIA SHIELD TV
- Plex Media Server version: 1.13.5154

**Comment by davilla** [comment]
NOTICE: script.plex: API: MDE: Multiple video streams, won't try to direct play

**Comment by Kalroth** [comment]
Hi Jelmerb, I've encountered the same issue, see issue #224.

**Comment by Jelmerb** [comment]
Hi Kalroth,

Thanks for clarifying the issue. I don’t think I can easily edit files on my SHIELD unless I have root access. I’ll wait for a fix.

**Comment by davilla** [comment]
fixed in testflight for v3.7.0

**Comment by Jelmerb** [comment]
Thanks, I didn't think about that. I masked the tokens in my log. How can you prevent this?

**Comment by davilla** [comment]
@pannal, yes. That fix has been in our v3.7.0 testflight for several weeks now. No regressions seen.
